airbyte-connector-builder-server/connector_builder/generated/models/dpath_extractor.py

```diff
@@ -19,13 +19,13 @@ class DpathExtractor(BaseModel):
 
     DpathExtractor - a model defined in OpenAPI
 
-        field_pointer: The field_pointer of this DpathExtractor.
+        field_path: The field_path of this DpathExtractor.
         config: The config of this DpathExtractor.
         decoder: The decoder of this DpathExtractor [Optional].
     """
 
-    field_pointer: List[AnyOfInterpolatedStringstring] = Field(alias="field_pointer")
+    field_path: List[AnyOfInterpolatedStringstring] = Field(alias="field_path")
```

Comment:
This file was autogenerated in December 2022. However, I can't find what is the process to re-generate this. I see [this](https://github.com/airbytehq/airbyte/blob/ca8cdc4/airbyte-connector-builder-server/src/main/openapi/openapi.yaml#L222) has been commented out but I don’t see any alternative solution provided. Also, the file used seems to be the previous json file and not `declarative_component_schema.yaml`. Based on the commit/issues associated, it seems there was a lot of struggle with code generation. Therefore, I have a couple of questions:
* Do we need code generation for the manifest in the Connector Server Builder?
* If so, should we clarify the state of the code generation in our README.md?
* If not, should we clean `openapi.yaml` file?